apiary.apib

```diff
@@ -1682,6 +1682,364 @@ asynchronously once the job is created.
             }
 
 
+# Group Factory API
+
+The Factory API allows you to interact with orders that have been placed and move them through the various
+stages of production. The typical workflow is as follows:
+
+<a href="https://github.com/unmadeworks/api-docs/assets/13105806/c638182b-ab00-4141-96de-e4fe77de2a1f">
+  <img src="https://github.com/unmadeworks/api-docs/assets/13105806/c638182b-ab00-4141-96de-e4fe77de2a1f" alt="Image showing the typical user journey through the e-commerce site" width="100%">
+</a>
+
+
+#### Factory Slugs
+
+All Factory API endpoints begin with `/v2/factories/<factory_slug>` where `factory_slug` is the unique identifier
+for each of your factory locations. These slugs will be provided to you by Unmade. Each API token provided to you
+can have access to **one or more individual factory locations**, depending on your specific requirements.
+
+#### Order States
+
+During production, orders progress through the following states:
+
+ - `new`: The order has not yet begun production
+ - `in_production`: The order has been accepted and is now in production
+ - `ready-to-ship`: The order has been produced and is awaiting shipping
+ - `shipped`: The order has left the factory
+ - `cancelled`: The order has been cancelled
+
+#### Job States
+
+After creation, the line items from a given order are split into one or more production jobs. These jobs represent
+one or more individual garments to be manufactured. Each job has its own production state, which are as follows:
+
+ - `new`: The job has not yet begun production
+ - `in_production`: The job has been accepted and is now in production
+ - `ready-to-ship`: The job has been produced and is awaiting shipping
+ - `shipped`: The job has left the factory
+ - `cancelled`: The job has been cancelled
```

Comment:
I'm using the Factory OMS UI states here to match the UI. I'm expecting to pass the filter querystring right down to Factory OMS and let Factory work it out, which may be slightly tricky since the [Job UI state logic](https://github.com/unmadeworks/factory/blob/46abeee899e8ef590d6560aa368275ceb5daf36e/unmade_factory/job/helpers.py#L8) is pretty wonky